src/content/docs/en/welcome.mdx

```diff
@@ -5,30 +5,32 @@ description: "This wiki is meant to be the extended documentation for both end u
 
 Welcome to the Ultramarine Wiki! This wiki is designed to help users get settled into Ultramarine Linux.
 
-Developer and Package Maintainer Documentation has moved to [Fyra Developer](https://developer.fyralabs.com). If you want to get started with contributing, head to the [Contributing Section](/en/contributing/contributorguide).
+Developer and Package Maintainer Documentation has moved to [Fyra Developer](https://developer.fyralabs.com).
+
+If you want to get started with contributing, head to the [Contributing Section](/en/contributing/contributorguide).
 
 On this page, you can read about the project, so if you want to become familiar with important concepts and our philosophy, read this first!
 
 If you want to install Ultramarine Linux, head to the [Getting Ultramarine](/en/installation/getting) section.
 
 ## General Philosophy
 
-Ultramarine Linux is a spiritual successor to Korora Linux, intending to make an operating system that "just works" for people of all levels of experience.
-
-To do this, we stick to a set of values:
+Ultramarine Linux's primary goal is to create a instantly useable experience while keeping up with innovations in the space, we represent this with a set of statements:
 
 ### Ultramarine is Pragmatic.
 
 Where Fedora tries to adhere to ideology (that’s perfectly fine), by shipping an (almost) entirely free system, Ultramarine works off the model of least resistance. That means we try to make it easy for the user, even if we have to ship some proprietary drivers and codecs that Fedora wouldn’t be comfy with out-of-box.
 
-### Ultramarine is Innovative.
+### Ultramarine is Progressive.
 
 We believe that recent innovations in the Linux space serve to greatly improve the user and developer experience. Technologies like Flatpak and xdg-portals allow developers to provide great experiences to our users more consistently. Work done upstream improves the experience for our users, and work done in the Ultramarine ecosystem; like accent colours, and Chromebook support, improves the larger Linux ecosystem. It's a win-win.
 
 ### Ultramarine is User-friendly.
 
 While it may seem trivial to more advanced users to tweak a few settings and install some packages from 3rd party repositories, it may take people who are new to Linux (or even just new to Fedora) hours to figure out what packages they need, and where to find them. Ultramarine allows users new and old to get up and running quickly.
 
+### Ultramarine is Accesible

```

Comment:
?

Reply:
new value we're adding, per flock talk, still need to write it out

Reply:
accessible as in a11y?

Reply:
that's part of it, a11y is part of making ultramarine accessible to as many people as possible. accessibility comes from access-ability the ability to access, we want ultramarine to be easy to access, this is done with hardware, a11y, etc etc